pkg/reconcilers/configmap.go

```diff
@@ -18,3 +19,29 @@ func ConfigMapReconcileField(desired, existing *v1.ConfigMap, fieldName string)
 	}
 	return updated
 }
+
+func RedisConfigMapReconcileField(desired, existing *v1.ConfigMap, fieldName string) bool {
+	updated := false
+
+	if existingVal, ok := existing.Data[fieldName]; !ok {
+		existing.Data[fieldName] = desired.Data[fieldName]
+		updated = true
+	} else {
+		existingString := existingVal
+
+		replicaOfString := "rename-command REPLICAOF \"\""
+		if !strings.Contains(existingString, replicaOfString) {
+			newString := existingString + "\n" + replicaOfString
+			existing.Data[fieldName] = newString
+			updated = true
+		}
+
+		slaveOfString := "rename-command SLAVEOF \"\""
+		if !strings.Contains(existingString, slaveOfString) {
+			newString := existingString + "\n" + slaveOfString
```

Comment:
I think this needs to be:
existingString = existingString + "\n" + <RENAME_COMMAND> in both, REPLICAOF and SLAVEOF. 
Otherwise we are going to need 2 reconciles for both to be applied if both are missing.